python/src/scipp/compat/mantid.py

```diff
@@ -619,10 +619,12 @@ def convert_EventWorkspace_to_data_array(ws,
                         shape=[n_event],
                         unit=unit,
                         dtype=sc.dtype.float64)
-    weights = sc.Variable(dims=['event'],
-                          unit=data_unit,
-                          values=np.ones(n_event, dtype=np.float32),
-                          variances=np.ones(n_event, dtype=np.float32))
+    weights = sc.broadcast(sc.Variable(value=1.0,
```

Comment:
How exactly does broadcast help with the memory profile?

Reply:
Instead of creating a numpy array of ones and making a copy into scipp, we directly allow in scipp.